packages/office-ui-fabric-react/src/components/HoverCard/HoverCard.base.tsx

```diff
@@ -161,6 +161,21 @@ export class HoverCardBase extends BaseComponent<IHoverCardProps, IHoverCardStat
     }
   }
 
+  private _getEventListenerElement(): HTMLElement | undefined {
+    const { eventListenerTarget } = this.props;
+
+    switch (typeof eventListenerTarget) {
```

Comment:
Since `eventListenerTarget` is typed as `HTMLElement` we shouldn't need all of this runtime type handling.